### DISCUSSION TITLE
How to get rid of the Maximization button

### DISCUSSION THREAD

**Discussion by shovelbox** [lepoco/wpfui#756] [Q&A] [answered]
How to get rid of the Maximization button
```

 <ui:TitleBar
     x:Name="TitleBar"
     Title="{Binding ViewModel.ApplicationTitle}"
     Grid.Row="0"
     CloseWindowByDoubleClickOnIcon="True">
     <ui:TitleBar.Icon>
         <ui:ImageIcon Source="pack://application:,,,/Assets/wpfui-icon-256.png" />
     </ui:TitleBar.Icon>
 </ui:TitleBar>

```

**Comment by Mithicor** [comment, marked as answer]
I am not sure about removing it, but you can disable it with the "CanMaximize" property.

**Reply by shovelbox** [reply]
How can disable it?

**Reply by Mithicor** [reply]
`<ui:TitleBar CanMaximize="False"/>`

When you hover the Maximize button, you will still be able to use "Snap layouts", but clicking the maximize button will do nothing.

**Reply by shovelbox** [reply]
<ui:TitleBar
     x:Name="TitleBar"
     ShowMinimize="False"
     ShowMaximize="False"/>
     😁

**Reply by Veritera** [reply]
If you got the answer mark is as answered.
